project_singlefamilydetached/singlefamilydetached_with_upgrades.yml

```diff
@@ -0,0 +1,1165 @@
+        lifetime: 15
+        reference_scenario: Reference 80% AFUE Furnace (gas, propane, oil)
+  - upgrade_name: ENERGY STAR Boiler (Gas, Propane, Oil)
+    options:
+      - option: HVAC System Heating Natural Gas|Gas Boiler, 96% AFUE
+        apply_logic:
+          or:
+            - HVAC System Heating Natural Gas|Gas Boiler, 72% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 76% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 80% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 85% AFUE
+        costs:
+          - value: 43
+            multiplier: Size, Heating System (kBtu/h)
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24
+        reference_scenario: Reference Boiler (gas, propane, oil)
+      - option: HVAC System Heating Propane|Propane Boiler, 96% AFUE
+        apply_logic:
+          or:
+            - HVAC System Heating Propane|Propane Boiler, 72% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 76% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 80% AFUE
+        costs:
+          - value: 43
+            multiplier: Size, Heating System (kBtu/h)
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24
+        reference_scenario: Reference Boiler (gas, propane, oil)
+      - option: HVAC System Heating Fuel Oil|Oil Boiler, 95% AFUE
+        apply_logic:
+          or:
+            - HVAC System Heating Fuel Oil|Oil Boiler, 72% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 76% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 80% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 90% AFUE
+        costs:
+           - value: 40
+             multiplier: Size, Heating System (kBtu/h)
+           - value: 3100
+             multiplier: Fixed (1)
+        lifetime: 24
+        reference_scenario: Reference Boiler (gas, propane, oil)
+  - upgrade_name: Reference Boiler (gas, propane, oil)
+    options:
+      - option: HVAC System Heating Natural Gas|Gas Boiler, 80% AFUE
+        apply_logic:
+          or:
+            - HVAC System Heating Natural Gas|Gas Boiler, 72% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 76% AFUE
+        costs:
+          - value: 21
+            multiplier: Size, Heating System (kBtu/h)
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24
+      - option: HVAC System Heating Propane|Propane Boiler, 80% AFUE
+        apply_logic:
+          or:
+            - HVAC System Heating Propane|Propane Boiler, 72% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 76% AFUE
+        costs:
+          - value: 21
+            multiplier: Size, Heating System (kBtu/h)
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24
+      - option: HVAC System Heating Fuel Oil|Oil Boiler, 80% AFUE
+        apply_logic:
```

Comment:
Need to add `- or:` layer, like the option just preceding this one

Reply:
Good catch.